audio/audio_driver.h

```diff
@@ -158,13 +167,84 @@ typedef struct audio_driver
    size_t (*buffer_size)(void *data);
 } audio_driver_t;
 
+/**
```

Comment:
Commented because I thought I might need similar flags for the mic driver; to do that, I had to understand what these did.